pkg/pillar/cmd/volumemgr/initialvolumestatusold_test.go

```diff
@@ -124,7 +124,7 @@ func TestParseAppRwVolumeName(t *testing.T) {
 	}
 	for testname, test := range testMatrix {
 		t.Logf("Running test case %s", testname)
-		dir, sha, uuid, purgeCounter := parseAppRwVolumeName(test.filename, test.isContainer)
+		dir, sha, uuid, purgeCounter := parseAppRwOldVolumeName(test.filename, test.isContainer)
```

Comment:
You also want to add test cases for parseAppRwVolumeName.

Reply:
Added unit tests in initialvolumestatus_test.go